src/utils/viewer/viewer.cpp

```diff
@@ -180,9 +176,13 @@ Viewer::Viewer(const Arguments& arguments)
   rgbSensorNode_->translate({0.0f, rgbSensorHeight, 0.0f});
   agentBodyNode_->translate({0.0f, 0.0f, 5.0f});
 
-  int width = viewportSize[0];
-  int height = viewportSize[1];
-  renderCamera_->setProjectionMatrix(width, height, znear_, zfar_, hfov_);
+  renderCamera_->setProjectionMatrix(viewportSize[0],  // width
+                                     viewportSize[1],  // height
```

Comment:
`.x()` and `.y()` instead of `[0]` and `[1]`? :) then you won't need to comment `// width` `// height`